conda_forge_tick/update_sources.py

```diff
@@ -616,20 +618,6 @@ def get_version(self, url: str) -> Optional[str]:
         return latest
 
 
-class LibrariesIO(VersionFromFeed):
```

Comment:
Why was this deleted?

Reply:
It doesn't seem to be used anywhere. Also, it uses a method `self.package_name()` which doesn't exist? Looking through the git history it seemed like maybe it was copy/pasted from a while ago and the code base had evolved since then without this class being updated (as it wasn't being used anywhere).

Happy to add it back though if I have this all wrong?

Reply:
You don't have it wrong per say, but let's keep the dead code around just in case someone who wanted it comes asking about it.